### PR TITLE
[PX-4857] Billing address isn't saved to the credit card for arta shipping

### PR DESCRIPTION
**Pull request by fladson**
This fixes the case where orders with Arta enabled and with "Billing and shipping address are the same" option enabled, weren't saving the address on CreditCard records.

_Before and after_
Screenshot 2022-02-02 at 14 54 37

### CODE REVIEW

File: src/v2/Apps/Order/Components/PaymentPicker.tsx [hunk 1]
```diff
@@ -520,6 +520,15 @@ export const PaymentPickerFragmentContainer = createFragmentContainer(
...
           }
+          ... on CommerceShipArta {
+            name
+            addressLine1
+            addressLine2
+            city
+            region
+            country
+            postalCode
+          }
```
Comment (pepopowitz) [inline src/v2/Apps/Order/Components/PaymentPicker.tsx:531]: I was initially unsure how adding fields to the query fragment would resolve this issue, but I think what is happening is the values that come back in the query [get added to the address that's sent to stripe](https://github.com/artsy/force/blob/8a1b817e2b85add7038e5776f0df57bbd4d53138/src/v2/Apps/Order/Components/PaymentPicker.tsx#L375-L380) -- do I understand that correctly?

**Review by pepopowitz — CHANGES_REQUESTED** [review]
Nice, I think it makes sense! My only blocking request is to either match .tool-versions to .nvmrc or remove it.

File: src/v2/Apps/Order/Components/PaymentPicker.tsx (hunk 1 above)
Comment (jpotts244) [reply src/v2/Apps/Order/Components/PaymentPicker.tsx:531]: Fascinating find!

File: .tool-versions [hunk 2]
```diff
@@ -0,0 +1 @@
+nodejs 12.20.2
```
[the remark this answers is not in the document]
Comment (fladson) [reply .tool-versions:1]: 🤔  I based that on the `package.json`, will remove it.

File: src/v2/Apps/Order/Components/PaymentPicker.tsx (hunk 1 above)
Comment (fladson) [reply src/v2/Apps/Order/Components/PaymentPicker.tsx:531]: yeah, that's pretty much it.

**Review by pepopowitz — APPROVED** [review]

**Pull request merged**